android/sdl_android/src/androidTest/java/com/smartdevicelink/test/Test.java

```diff
@@ -791,6 +791,7 @@ public void onVoiceCommandSelected() {
         GENERAL_PERMISSIONITEM.setRpcName(GENERAL_STRING);
         GENERAL_PERMISSIONITEM.setHMIPermissions(GENERAL_HMIPERMISSIONS);
         GENERAL_PERMISSIONITEM.setParameterPermissions(GENERAL_PARAMETERPERMISSIONS);
+		GENERAL_PERMISSIONITEM.setEncryptionRequirement(GENERAL_BOOLEAN);
```

Comment:
Very minor request, could we please match the spacing from the line above?

Reply:
Done